pkg/dns/aws/dns.go

```diff
@@ -242,83 +242,101 @@ func (m *Provider) change(record *iov1.DNSRecord, zone configv1.DNSZone, action
 		return fmt.Errorf("unsupported record type %s", record.Spec.RecordType)
 	}
 	// TODO: handle >0 targets
```

Comment:
Delete the TODO.  There are a few other TODOs you can delete as well; check `git grep -e 'TODO: handle >0 targets'`.

Reply:
done .. do we need to fix https://github.com/openshift/cluster-ingress-operator/blob/master/test/e2e/operator_test.go#L438 ?

Reply:
No, I don't think that TODO makes sense—we should just delete it.